CHANGELOG.md

```diff
@@ -28,7 +28,7 @@ Full documentation for Omniperf is available at [https://rocm.docs.amd.com/proje
 
 ### Changes
 
-  * new option to force hardware target via `OMNIPERF_ARCH_OVERRIDE` global (#370)
+  * new option to force hardware target via `ROCPROFCOMPUTE_ARCH_OVERRIDE` global (#370)
```

Comment:
```suggestion
  * new option to force hardware target via `OMNIPERF_ARCH_OVERRIDE` global (#370)
```
The CHANGELOG probably doesn't need changing, because at the time is _was_ Omniperf.